src/Commands/Generators/Views/usermodel.tpl.php

```diff
@@ -0,0 +1,20 @@
+<@php
+
+declare(strict_types=1);
+
+namespace {namespace};
+
+use CodeIgniter\Shield\Models\UserModel;
+
+class {class} extends UserModel
+{
+    protected function initialize(): void
+    {
+        // Merge properties with parent
+        $this->allowedFields = array_merge($this->allowedFields, [
+            // Add here your custom fields
+            // 'first_name',
+        ]);
```

Comment:
Since we're on PHP 7.4+, we can use array unpacking.
```suggestion
        // Merge properties with parent
        $this->allowedFields = [
            ...$this->allowedFields,
            // Add here your custom fields
            // 'first_name',
        ];
```